packages/edit-site/src/components/sidebar-edit-mode/global-styles-sidebar.js

```diff
@@ -57,6 +78,20 @@ export default function GlobalStylesSidebar() {
 	}, [ shouldClearCanvasContainerView ] );
 
 	const { setIsListViewOpened } = useDispatch( editSiteStore );
+	const { goTo } = useNavigator();
+	const loadRevisions = () => {
```

Comment:
I've moved the revision icon in the styles panel header here to make it easier to work with style book vars. 

That's the theory 😄

Reply:
Actually, this change might not be needed. Might be a okay for a follow up as it consolidates styles actions.

Reply:
I'll do this in a follow up:

- https://github.com/WordPress/gutenberg/pull/57034